### PR TITLE
Add spec for WebResourceRequested.md

### PR DESCRIPTION
**Pull request by champnic**
This is a review for the new WebResourceRequested API.

### CODE REVIEW

**Comment by champnic** [comment]
Testing general discussion comment.

File: specs/WebResourceRequested.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+<!-- USAGE
+  * Fill in each of the sections (like Background) below
+  * Wrap code with `single line of code` or ```code block```
+  * Before submitting, delete all <!-- TEMPLATE marked comments in this file,
+    and the following quote banner:
```
Comment (champnic) [inline specs/WebResourceRequested.md:5]: Comment about this line. Must change.

**Comment by david-risney** [comment]
Was this a test PR? Is it OK to close it or rename it so I don't accidentally go to review it in my list =) ?

**Review by david-risney — CHANGES_REQUESTED** [review]
Just the one question if this is intended to be a test PR and if so can we close it or rename it away from "Add spec for WebResourceRequested" so I don't click on it in my PR list.

**Comment by champnic** [comment]
Ha, forgot about this. Thanks for reminding me :)

**Pull request closed without merging**